cmd/manager/main.go

```diff
@@ -56,12 +57,12 @@ func main() {
 		os.Exit(1)
 	}
 
-	ll := log.WithValues("namespace", namespace)
+	log.UseLogger(log.WithValues("namespace", namespace))
```

Comment:
is this just a 1:1 replacement of the `log.WithValues` pattern? (it doesn't extend past the scope of the file its in)

Reply:
No, this tells the global logger to use these fields

Reply:
i'm not sure we want to set this here then... the operator watches all namespaces, unless we're just printing the namespace that the operator is currently in? is the idea that later on it will be correctly overwritten?